packages/styling/src/styles/DefaultSpacing.ts

```diff
@@ -0,0 +1,9 @@
+import { ISpacing } from '../interfaces/index';
+
+export const DefaultSpacing: ISpacing = {
```

Comment:
Is it possible to keep these in experiments for now? I know the chance of someone else using these is small, but it is a public contract, so we'd be committing to this convention.

Reply:
The interface is marked as internal and I'm not sure we can avoid. I did the same with ITypography.